### PR TITLE
Add optional tags to create-service command

### PR DESCRIPTION
**Pull request by Gerg**
Tracker Story: https://www.pivotaltracker.com/story/show/61861194

### CODE REVIEW

**Comment by jberkhahn** [comment]
Hey @Gerg 
It looks like we've updated the translation files for this since you submitted the PR. Could you rebase on master to fix the merge conflicts?
Other than that this looks good.

**Comment by rmasand** [comment]
We've done a rebase, should be an easier merge now.

File: cf/i18n/resources/fr_FR.all.json [hunk 1]
```diff
@@ -4540,6 +4540,11 @@
       "modified": false
    },
    {
+      "id": "User provided tags",
+      "translation": "User provided tags",
```
Comment (maximilien) [inline cf/i18n/resources/fr_FR.all.json]: Translation: `"Étiquettes fourni par l'utilisateur"`

File: cf/i18n/resources/fr_FR.all.json [hunk 2]
```diff
@@ -580,8 +580,8 @@
...
+      "id": "CF_NAME bind-service APP_NAME SERVICE_INSTANCE [-c PARAMETERS_AS_JSON]\n\n  Optionally provide service-specific configuration parameters in a valid JSON object in-line.\n  CF_NAME bind-service APP_NAME SERVICE_INSTANCE -c '{\"name\":\"value\",\"name\":\"value\"}'\n\n  Optionally provide a file containing service-specific configuration parameters in a valid JSON object. The path to the parameters file can be an absolute or relative path to a file.\n  CF_NAME bind-service APP_NAME SERVICE_INSTANCE -c PATH_TO_FILE\n\n   Example of valid JSON object:\n   {\n     \"cluster_nodes\": {\n        \"count\": 5,\n        \"memory_mb\": 1024\n      }\n   }\n\nEXAMPLE:\n\u0009Linux/Mac:\n\u0009\u0009CF_NAME bind-service myapp mydb -c '{\"permissions\":\"read-only\"}'\n\n\u0009Windows Command Line\n\u0009\u0009CF_NAME bind-service myapp mydb -c \"{\\\"permissions\\\":\\\"read-only\\\"}\"\n\n\u0009Windows PowerShell\n\u0009\u0009CF_NAME bind-service myapp mydb -c '{\\\"permissions\\\":\\\"read-only\\\"}'\n\u0009\n\u0009CF_NAME bind-service myapp mydb -c ~/workspace/tmp/instance_config.json",
```
Comment (maximilien) [inline cf/i18n/resources/fr_FR.all.json]: This is a mess to translate (any language). I wonder if this string could have been splitted into smaller ones?

**Comment by rmasand** [comment]
Because #474 was merged already, this PR now just includes some formatting changes to the help text and translations.

**Comment by jberkhahn** [comment]
Nice! Merged.

**Pull request merged**